lib/features/favorite_list/favorite_connector.dart

```diff
@@ -0,0 +1,20 @@
+import 'package:async_redux/async_redux.dart';
+import 'package:flutter/material.dart';
+import 'package:pokedex/features/favorite_list/favorite_pokemon.dart';
+import 'package:pokedex/features/favorite_list/favorite_vm.dart';
+import 'package:pokedex/state/app_state.dart';
+
+class FavoriteConnector extends StatelessWidget {
+  const FavoriteConnector({Key? key}) : super(key: key);
+
+  static const String route = '/favorites';
+  static const String name = 'favorites';
```

Comment:
Are you using this for routing? if yes can you be descriptive with the naming and also I think you can reuse the existing favorites from the constants 🤔

Reply:
Yes it is used for routing but I think the name is descriptive as it is once you used it (ex. PokemonListConnector.route) I removed the constants and instead made a static const of them to limit their access if the parent class is used.

Reply:
@yohancai route is fine what i mean here is the name.

Reply:
should i change name to pokemonName and favoriteName?